### PR TITLE
[Validator] Fix CS in constraint tests

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
fix 4.4 right?

**Comment by derrabus** [comment]
On 4.4, these tests don't exist. On 5.2, all of these are wrapped in `eval()` statements because of the named argument calls. So we only need to fix that on 6.0.

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

File: src/Symfony/Component/Validator/Tests/Constraints/ChoiceValidatorTest.php [hunk 1]
```diff
@@ -112,8 +112,8 @@ public function provideConstraintsWithCallbackFunction(): iterable
         ])];
         yield 'doctrine style, static method' => [new Choice(['callback' => [__CLASS__, 'staticCallback']])];
         yield 'named arguments, namespaced function' => [new Choice(callback: 'Symfony\Component\Validator\Tests\Constraints\choice_callback')];
-        yield 'named arguments, closure' => [new Choice(callback: fn () => ["foo", "bar"])];
-        yield 'named arguments, static method' => [new Choice(callback: ["Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest", "staticCallback"])];
+        yield 'named arguments, closure' => [new Choice(callback: fn () => ['foo', 'bar'])];
+        yield 'named arguments, static method' => [new Choice(callback: ["Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest", 'staticCallback'])];
```
Comment (OskarStark) [inline src/Symfony/Component/Validator/Tests/Constraints/ChoiceValidatorTest.php]:
```suggestion
        yield 'named arguments, static method' => [new Choice(callback: ['Symfony\Component\Validator\Tests\Constraints\ChoiceValidatorTest', 'staticCallback'])];
```
?

Comment (derrabus) [reply src/Symfony/Component/Validator/Tests/Constraints/ChoiceValidatorTest.php]: Apparently, CS Fixer did not pick that up. 🤔

**Pull request merged**